README.md

```diff
@@ -54,6 +54,12 @@ any of the following keys:
 Determines if the token secret for the user should be stored in a cookie
 or in `req.session`. Defaults to `false`.
 
+This implements the [Double Submit Cookie Pattern](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#double-submit-cookie) and is
+easy to implement, but you should be sure that the caveats do not apply to you.
+
+In particular, the client still needs to send the CSRF token in a separate
```

Comment:
What does this part mean? I didn't see that on the OWASP page, and not sure what that means in this module, either, as this module does not include the token in a cookie (just the secret is in the cookie). As the author, I'm not even sure what this means in order to follow, so unsure other users will too, which is why I'm asking if it can be clarified.

Reply:
I saw a few issues about people adding middleware for sending the token as a cookie, thereby completely invalidating the concept of CSRF.

I wanted to warn against that. How about

> In particular, the CSRF token itself should not be put in a cookie, since that would remove the CSRF protection. Instead, it should be passed to the client so the client can present it to validate the request.

?

Reply:
How does putting into a cookie compromise csrf protection, specifically? This module does not read the value back from a cookie, only from a separate request parameter. Even if the user puts in in a cookie, this module will not honor it.

Reply:
ah - I misread the configuration, I was under the impression that you could provide the token in a cookie and the middleware would read it from there as well. I'll remove this paragraph then.

Reply:
Gotcha. This only stores the secret in the cookie. The reason why you see people putting the token in a cookie is because csrf plugins for things like angular will read the token from there client side, but then still include it as a request parameter on submit.